support/src/main/java/org/jenkinsci/plugins/workflow/support/visualization/table/FlowGraphTable.java

```diff
@@ -250,6 +251,10 @@ boolean isEnd() {
             return node instanceof BlockEndNode;
         }
 
+        boolean isExecuted() {
+            return node.getAction(NotExecutedNodeAction.class) == null;
```

Comment:
Right, the won't have the `NotExecutedNodeAction`, so will appear as they always did.